package.json

```diff
@@ -2,8 +2,8 @@
   "name": "spectacle",
   "version": "5.5.0",
   "description": "ReactJS Powered Presentation Framework",
-  "main": "lib/index.js",
-  "module": "es/index.js",
```

Comment:
There is still `es/` babel output in this PR. Let's leave it until its time to remove.